### PR TITLE
More true/false normalization

### PR DESCRIPTION
**Pull request by felixfontein**
##### SUMMARY
More boolean normalization done with https://github.com/ansible-community/community-team/issues/60#issuecomment-1456867303.

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
various

### CODE REVIEW

File: plugins/modules/aix_devices.py [hunk 1]
```diff
@@ -108,7 +108,7 @@
     device: en1
     attributes:
       mtu: 900
-      arp: off
+      arp: false
```
Comment (kristianheljas) [inline plugins/modules/aix_devices.py:111]: I think this should be string, just like in the [integration test](https://github.com/ansible-collections/community.general/pull/6152/files#diff-13e125ace2da722b0ad5442bbcfbab8c3079eb02c553dd0c65316775de78c8e6R64) (or if i'm wrong, the integration test needs a change)
```suggestion
      arp: 'off'
```

**Review by kristianheljas — CHANGES_REQUESTED** [review]

File: plugins/modules/aix_devices.py (hunk 1 above)
Comment (felixfontein) [reply plugins/modules/aix_devices.py]: You are right, this should (very likely) be a string. Thanks!

**Review by kristianheljas — APPROVED** [review]
LGTM!

**Comment by felixfontein** [comment]
@kristianheljas thanks for reviewing this!

**Pull request merged**